phoenix-core/src/main/java/org/apache/phoenix/util/UpgradeUtil.java

```diff
@@ -1244,8 +1244,18 @@ public static void moveOrCopyChildLinks(PhoenixConnection oldMetaConnection, Map
                         QueryServices.MOVE_CHILD_LINKS_DURING_UPGRADE_ENABLED,
                         QueryServicesOptions.DEFAULT_MOVE_CHILD_LINKS_DURING_UPGRADE_ENABLED);
 
-        try (org.apache.hadoop.hbase.client.Connection moveChildLinkConnection =  getHBaseConnection(options);
-            Table sysCatalogTable = moveChildLinkConnection.getTable(TableName.valueOf(SYSTEM_CATALOG_NAME))) {
+        Configuration conf = oldMetaConnection.getQueryServices().getConfiguration();
+
+        ReadOnlyProps readOnlyProps = oldMetaConnection.getQueryServices().getProps();
+        TableName sysCat = SchemaUtil.getPhysicalTableName(SYSTEM_CATALOG_NAME, readOnlyProps);
+        TableName sysChildLink = SchemaUtil.getPhysicalTableName(SYSTEM_CHILD_LINK_NAME, readOnlyProps);
+
+        LOGGER.info(String.format("SYSTEM CATALOG tabled use for copying child links: %s", sysCat.toString()));
```

Comment:
Once the testing is done, you may want remove this, or change it to DEBUG level.